scripts/kani-top-100.sh

```diff
@@ -34,89 +34,7 @@ SELF_SCRIPT=$0
-https://github.com/RustCrypto/traits
-https://github.com/RustCrypto/utils
-https://github.com/SergioBenitez/version_check
-https://github.com/SimonSapin/rust-std-candidates
-https://github.com/alexcrichton/cc-rs
-https://github.com/alexcrichton/cfg-if
-https://github.com/alexcrichton/toml-rs
-https://github.com/bitflags/bitflags
-https://github.com/bluss/arrayvec
-https://github.com/bluss/either
-https://github.com/bluss/indexmap
-https://github.com/bluss/scopeguard
-https://github.com/chronotope/chrono
-https://github.com/clap-rs/clap
-https://github.com/contain-rs/vec-map
-https://github.com/crossbeam-rs/crossbeam
-https://github.com/cryptocorrosion/cryptocorrosion
-https://github.com/cuviper/autocfg
-https://github.com/dguo/strsim-rs
-https://github.com/dtolnay/anyhow
-https://github.com/dtolnay/itoa
-https://github.com/dtolnay/proc-macro-hack
-https://github.com/dtolnay/proc-macro2
-https://github.com/dtolnay/quote
-https://github.com/dtolnay/ryu
-https://github.com/dtolnay/semver
-https://github.com/dtolnay/syn
-https://github.com/dtolnay/thiserror
-https://github.com/env-logger-rs/env_logger
-https://github.com/fizyk20/generic-array.git
-https://github.com/hyperium/h2
-https://github.com/hyperium/http
-https://github.com/hyperium/hyper
-https://github.com/marshallpierce/rust-base64
-https://github.com/matklad/once_cell
-https://github.com/mgeisler/textwrap
-https://github.com/ogham/rust-ansi-term
-https://github.com/paholg/typenum
-https://github.com/retep998/winapi-rs
-https://github.com/rust-itertools/itertools
-https://github.com/rust-lang-nursery/lazy-static.rs
-https://github.com/rust-lang/backtrace-rs
-https://github.com/rust-lang/futures-rs
-https://github.com/rust-lang/hashbrown
-https://github.com/rust-lang/libc
-https://github.com/rust-lang/log
-https://github.com/rust-lang/pkg-config-rs
-https://github.com/rust-lang/regex
-https://github.com/rust-lang/socket2
-https://github.com/rust-num/num-integer
-https://github.com/rust-num/num-traits
-https://github.com/rust-random/getrandom
-https://github.com/rust-random/rand
-https://github.com/seanmonstar/httparse
-https://github.com/seanmonstar/num_cpus
-https://github.com/serde-rs/json
-https://github.com/serde-rs/serde
-https://github.com/servo/rust-fnv
-https://github.com/servo/rust-smallvec
-https://github.com/servo/rust-url
-https://github.com/servo/unicode-bidi
-https://github.com/softprops/atty
-https://github.com/steveklabnik/semver-parser
-https://github.com/taiki-e/pin-project-lite
-https://github.com/time-rs/time
-https://github.com/tokio-rs/bytes
-https://github.com/tokio-rs/mio
-https://github.com/tokio-rs/slab
-https://github.com/tokio-rs/tokio
-https://github.com/unicode-rs/unicode-normalization
-https://github.com/unicode-rs/unicode-segmentation
-https://github.com/unicode-rs/unicode-width
-https://github.com/unicode-rs/unicode-xid
-https://github.com/withoutboats/heck'
+HARD_CODED_TOP_100_CRATES_AS_OF_2022_6_17=$(cat $SELF_DIR/../tests/large-crawl/crates-top-100.txt)
```

Comment:
Can you make this an argument instead?

Reply:
Argument will be a bit difficult b/c of the way this script calls itself with different arguments during the actual work step. 

I did STDIN instead, but I am willing to do harder modifications if you strongly want arguments.

Reply:
Implemented request in b528facff1604454f2a1326e842e1da1b5b61cab.